### PR TITLE
Updated storybook favicon

### LINKED ISSUE #9754: Update the favicon of Storybook Setup with Ethereum favicon
Hey, I noticed the storybook setup does not have any favicon. Maybe we can use the same favicon we use on the ethereum.org website for the storybook setup as well. Let me know If this seems relevant, I can make a quick PR to fix this.

### PR DESCRIPTION
**Pull request by MukulKolpe**
## Description

Updated storybook favicon with Ethereum diamond icon (/src/assets/eth-home-icon.png).

## Related Issue

Fixes #9754

### CODE REVIEW

**Comment by MukulKolpe** [comment]
image

Hi, @TylerAPfledderer @corwintines. I've attached a screenshot here for reference. Let me know if there's anything to change.

**Comment by TylerAPfledderer** [comment]
Hmm the icon looks bloated. Could the same file that use for the site not be used? (I don't think it matters though)

**Comment by MukulKolpe** [comment]
Yes @TylerAPfledderer, Here's a screenshot with the favicon which is used on the website (/src/assets/favicon.png).

image

Should we go with `/src/assets/eth-home-icon.png` or `/src/assets/favicon.png` @TylerAPfledderer @corwintines?

File: .storybook/manager.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import { addons } from "@storybook/addons"
 import theme from "./theme"
+import favicon from "../src/assets/eth-home-icon.png"
```
Comment (corwintines) [inline .storybook/manager.js:3]:
```suggestion
import favicon from "../src/assets/favicon.png"
```

Agree with discussion, I think we might as well just use the favicon asset in this case. Other than that looks good to go

**Comment by MukulKolpe** [comment]
Awesome! Thank you @TylerAPfledderer, @corwintines 🎉

**Review by corwintines — APPROVED** [review]
Thanks @MukulKolpe!

**Pull request merged**